src/paymentservice/opentelemetry.js

```diff
@@ -1,41 +1,72 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-const opentelemetry = require("@opentelemetry/sdk-node")
-const {getNodeAutoInstrumentations} = require("@opentelemetry/auto-instrumentations-node")
-const {OTLPTraceExporter} = require('@opentelemetry/exporter-trace-otlp-grpc')
-const {OTLPMetricExporter} = require('@opentelemetry/exporter-metrics-otlp-grpc')
-const {PeriodicExportingMetricReader} = require('@opentelemetry/sdk-metrics')
-const {alibabaCloudEcsDetector} = require('@opentelemetry/resource-detector-alibaba-cloud')
-const {awsEc2Detector, awsEksDetector} = require('@opentelemetry/resource-detector-aws')
-const {containerDetector} = require('@opentelemetry/resource-detector-container')
-const {gcpDetector} = require('@opentelemetry/resource-detector-gcp')
-const {envDetector, hostDetector, osDetector, processDetector} = require('@opentelemetry/resources')
+const opentelemetry = require('@opentelemetry/sdk-node');
+const {
+	getNodeAutoInstrumentations,
+} = require('@opentelemetry/auto-instrumentations-node');
+const {
+	OTLPTraceExporter,
+} = require('@opentelemetry/exporter-trace-otlp-grpc');
+const {
+	OTLPMetricExporter,
+} = require('@opentelemetry/exporter-metrics-otlp-grpc');
+const { PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics');
+const {
+	alibabaCloudEcsDetector,
+} = require('@opentelemetry/resource-detector-alibaba-cloud');
+const {
+	awsEc2Detector,
+	awsEksDetector,
+} = require('@opentelemetry/resource-detector-aws');
+const {
+	containerDetector,
+} = require('@opentelemetry/resource-detector-container');
+const { gcpDetector } = require('@opentelemetry/resource-detector-gcp');
+const {
+	envDetector,
+	hostDetector,
+	osDetector,
+	processDetector,
+} = require('@opentelemetry/resources');
+
+const { PinoInstrumentation } = require('@opentelemetry/instrumentation-pino');
```

Comment:
Can you reformat the require statements to all be 1 line each, similar to how we have this done for other JS/TS files

Reply:
Okay